src/Http/Http.Abstractions/src/Validation/ValidateContext.cs

```diff
@@ -60,41 +61,139 @@ public sealed class ValidateContext
     /// </summary>
     public int CurrentDepth { get; set; }
 
+    /// <summary>
+    /// Gets or sets the JSON serializer options to use for property name formatting.
+    /// When set, property names in validation errors will be formatted according to the
+    /// PropertyNamingPolicy and JsonPropertyName attributes.
+    /// </summary>
+    public JsonSerializerOptions? SerializerOptions { get; set; }
```

Comment:
Let's remove this public API. Instead, use an internal type and derive the SerializerOptions by looking inside `ValidationContext.ServiceProvider`.